seed/data_importer/match.py

```diff
@@ -256,6 +258,10 @@ def inclusive_match_and_merge(unmatched_state_ids, org, StateClass):
     # Collapse groups of matches found in the previous step into 1 -State per group
     merges_within_file = 0
     priorities = Column.retrieve_priorities(org)
+    total_ids = len(matched_id_groups)
+    batch = int(total_ids/5)
+    batches = [batch*i for i in range(5)]
```

Comment:
Open to suggestions on naming convention. If there were 200 ids`batch` is 1/5th (`20`), and `batches` are (`[0, 40, 80, 120, 160]`)